### ISSUE TITLE
custom-style in HTML Import breaks IE11 rendering

### ISSUE THREAD

**Issue by brettpostin** [webcomponents/shadycss#112]
**TL;DR**

Using `<custom-style>` inside a HTML Import does not work and seemingly breaks all other rendering in IE11.

**Full Description**

I have a HTML Import (bundle), and within that bundle I have multiple elements along with a style module. The content of this style module needs to be accessed from both the document scope and in various element shadowdoms. 

**my-bundle.html**
```
<dom-module id="x-stylemodule">
  <template>
    <style>
      body {
        background-color: red;
      }
    </style>
  </template>
</dom-module>

<dom-module id="x-example">
  <template>
    Hello World
  </template>
  <script>
    Polymer({
      is: 'x-example'
    });
  </script>
</dom-module>

<custom-style>
    <style is="custom-style" include="x-stylemodule"></style>
</custom-style>
```

**index.html**
```
...
<link rel="import" href="my-bundle.html" />
<x-example></x-example>
...
```

This all works fine in Chrome, Firefox and Edge. "Hello World" is displayed and the background is red.

However in IE11 there is nothing but a blank white screen. No errors or console logs, and the `x-example` "Hello World" does not get rendered.

If I move the `<custom-style>` declaration out into `index.html` it works as expected. I want to avoid this however as it means any consumers of the bundle need to **manually** register the style module with the document scope.

Is the `<custom-style>` wrapper element restricted to the main document or should it be possible to register document level styles within a HTML Import?

**Comment by valdrinkoshi** [comment]
Hi @brettpostin, this should have been resolved in https://github.com/webcomponents/shadycss/pull/109, but hasn't been released yet. Can you try using `shadycss#master` to see if the issue is solved?

**Comment by valdrinkoshi** [comment]
This seems a very weird behavior on IE11 where `load` or `error` events are not triggered for an empty style element, e.g. `<style></style>`, but they are for `<style> </style>` (one text node as child). So the fix seems to be literally adding a space in that style, believe it or not :D
We should move this issue to HTMLImports, since that's the polyfill that should handle this case

**Comment by brettpostin** [comment]
Your workaround solves the issue for us in the meantime, thanks!
